tests/foreman/api/test_contentview.py

```diff
@@ -396,11 +350,10 @@ def test_positive_publish_1(self):
         @Feature: ContentView
 
         """
-        content_view = entities.ContentView()
-        content_view.id = content_view.create_json()['id']
+        content_view = entities.ContentView().create()
         for _ in range(REPEAT):
             content_view.publish()
-        self.assertEqual(len(content_view.read_json()['versions']), REPEAT)
+        self.assertEqual(len(content_view.read().version), REPEAT)
```

Comment:
I've been wondering about this. On one hand, it makes a bit of intuitive sense for us to use the word "versions". But on the other hand, all of NailGun's `OneTo*Field`s are named in the singular. (e.g. `ContentView.repository` is a `OneToManyField` accessed as `content_view.repository[0]`) Naming this field "version" instead of "versions" is in keeping with the singular naming scheme. What do you think?